### PR TITLE
Add missing rdb metrics

### PR DESCRIPTION
**Pull request by vin01**
these metrics have been added in 7.0 and are very useful in debugging, monitoring.

Reference: https://github.com/redis/redis/commit/56fa48ffc136b7540f8144562d8aa2a13b8a3fa7

### CODE REVIEW

File: exporter/exporter.go [hunk 1]
```diff
@@ -189,7 +189,10 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
 			"rdb_last_bgsave_status":       "rdb_last_bgsave_status",
 			"rdb_last_bgsave_time_sec":     "rdb_last_bgsave_duration_sec",
 			"rdb_current_bgsave_time_sec":  "rdb_current_bgsave_duration_sec",
+                        "rdb_saves":                    "rdb_saves",
 			"rdb_last_cow_size":            "rdb_last_cow_size_bytes",
+                        "rdb_last_load_keys_expired":   "rdb_last_load_keys_expired",
```
Comment (oliver006) [inline exporter/exporter.go:194]: rdb_last_load_expired_keys

File: exporter/exporter.go [hunk 2]
```diff
@@ -189,7 +189,10 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
...
+                        "rdb_last_load_keys_loaded":    "rdb_last_load_keys_loaded",
```
Comment (oliver006) [inline exporter/exporter.go:195]: rdb_last_load_loaded_keys

**Review by oliver006 — COMMENTED** [review]
Thanks for the PR!

Let's give thee metrics a bit more prometheus-friendly names (trailing unit like "keys" or "-total) and then this should be good to go.

File: exporter/exporter.go [hunk 3]
```diff
@@ -189,7 +189,10 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
 			"rdb_last_bgsave_status":       "rdb_last_bgsave_status",
 			"rdb_last_bgsave_time_sec":     "rdb_last_bgsave_duration_sec",
 			"rdb_current_bgsave_time_sec":  "rdb_current_bgsave_duration_sec",
+                        "rdb_saves":                    "rdb_saves",
```
[the remark this answers is not in the document]
Comment (vin01) [reply exporter/exporter.go]: sounds good, updated 👍

**Comment by oliver006** [comment]
Thanks for the PR!

**Comment by oliver006** [comment]
Released as https://github.com/oliver006/redis_exporter/releases/tag/v1.58.0

**Pull request merged**